docker/data-source-services/docker-compose.yml

```diff
@@ -39,6 +39,22 @@ services:
       - MYSQL_PASSWORD=mariadb_pass
     networks:
       - st-internal
+  minio:
+    hostname: s3.docker
+    image: minio/minio:latest
+    container_name: sourcetoad_minio
+    ports:
+      - "9000:9000"
+      - "9090:9090"
+    volumes:
+      - ./storage/data:/data
+    environment:
+      - MINIO_ROOT_USER=root
+      - MINIO_ROOT_PASSWORD=password
+    command: server /data/ --console-address :9090
+    restart: unless-stopped
```

Comment:
It was used in the set up I was using on a former contract, but I never actually had it trigger. It may have been there for a reason specific to their environment.  I feel pretty confident it can be removed.